lib/Command/RecreateMasterKey.php

```diff
@@ -204,24 +151,19 @@ protected function execute(InputInterface $input, OutputInterface $output) {
 
 				$output->writeln("Waiting for creating new masterkey\n");
 
-				$this->keyManager->setPublicShareKeyIDAndMasterKeyId();
-
-				$output->writeln("New masterkey created successfully\n");
-
-				$this->appConfig->setValue('encryption', 'enabled', 'yes');
-				$this->appConfig->setValue('encryption', 'useMasterKey', '1');
+				if ($this->encDecAllFactory->getEncryptAllObj()->createMasterKey($input, $output) === false) {
```

Comment:
Just create one `EncryptAll` object. You're creating one here and another one below.

Reply:
Done.